src/components/heading/heading.spec.tsx

```diff
@@ -41,6 +41,33 @@ describe("Heading", () => {
     });
   });
 
+  describe("when headingType prop is provided", () => {
+    const HeadingOption1: HeadingTypes = "h1";
+    const HeadingOption2: HeadingTypes = "h2";
+    const HeadingOption3: HeadingTypes = "h3";
+    const HeadingOption4: HeadingTypes = "h4";
+    const HeadingOption5: HeadingTypes = "h5";
+    const HeadingOption6: HeadingTypes = "h6";
+
+    it.each([
+      HeadingOption1,
+      HeadingOption2,
+      HeadingOption3,
+      HeadingOption4,
+      HeadingOption5,
+      HeadingOption6,
+    ])(
+      "HTML section heading element is correct when headingType is %s",
```

Comment:
What do you think about amending the test description to the following so its clearer which element the test is referring to? This is minor so happy to accept otherwise 👍🏼 
```suggestion
      "title is rendered as the correct heading element when headingType is %s",
```

Reply:
Yeah I agree, I'll change 👍